stix_shifter_modules/splunk/stix_translation/query_constructor.py

```diff
@@ -190,9 +187,32 @@ def translate(self, expression):
                 self.translate(expression.expr2)
             )
 
+    def _field_severity(self, value):
+        """
+        check for severity and convert input value to
+        informational,low,medium,high,critical
+        param value
+        return value(str)
+        """
+        value = int(value)
+        if 1 <= value <= 20:
+            value = str("informational")
+        elif 21 <= value <= 40:
+            value = str("low")
+        elif 41 <= value <= 60:
+            value = str("medium")
+        elif 61 <= value <= 80:
+            value = str("high")
+        elif 81 <= value <= 100:
+            value = str("critical")
+        else:
+            raise NotImplementedError('only 1-100 integer values are supported with severity field')
+        return value
+        
     def _build_comparison(self, expression, object_scoping, field_mapping):
         comparator = self._lookup_comparison_operator(self, expression.comparator)
-
+        if field_mapping == 'severity':
+            expression.value = self._field_severity(expression.value)
```

Comment:
Do other connectors map severity like this?  According to https://github.com/opencybersecurityalliance/stix-extensions/blob/main/2.0/x-ibm-finding.md it should be a number from 1 to 100.

Reply:
Oh, I read it backwards.  Thank you for the explanation.